compiler/circle-mpqsolver/README.md

```diff
@@ -0,0 +1,68 @@
+# circle-mpqsolver
+_circle-mpqsolver_ provides light-weight methods for finding a high-quality mixed-precision model 
+within a reasonable time.
+
+## Methods
+
+### Bisection
+A model is split into two parts: front and back. One of them is quantized in uint8 and another in 
+int16. The precision of front and back is determined by our proxy metric, upperbound of total layer 
+errors. (See #10170 (comment) for more details)
```

Comment:
> #10170 (comment)

This does not show to a link and `comment isn't point to which comment to the issue.
It would be better to (1) fix this to a real link so that reader can jump or (2) insert the comment here and add a reference link to the source.

Reply:
Ahhh... sorry. I'll fix it.